### DISCUSSION TITLE
Hovered component in canvas isn't working exactly

### DISCUSSION THREAD

**Discussion by khanhleemtp** [GrapesJS/grapesjs#4322] [Q&A]
![Screenshot from 2022-05-12 11-52-56](https://user-images.githubusercontent.com/58402281/167998922-5cc2475e-22e1-4748-9b0c-f1190d7e3ace.png)

I have function, i click arrow button, canvas is toggle. After i toggle canvas and i hovered component in canvas isn't working exactly. I think frame is not re caculate when i toggle

**Comment by artf** [comment]
Sorry but there is no way to help without a reproducible demo
